common/src/test/java/com/netflix/conductor/common/events/TestEventHandler.java

```diff
@@ -0,0 +1,36 @@
+package com.netflix.conductor.common.events;
+
+import com.netflix.conductor.common.metadata.events.EventHandler;
+import com.netflix.conductor.common.metadata.workflow.WorkflowTask;
+import org.junit.Test;
+
+import javax.validation.ConstraintViolation;
+import javax.validation.Validation;
+import javax.validation.Validator;
+import javax.validation.ValidatorFactory;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class TestEventHandler {
```

Comment:
Follow test naming convention, name this `EventHandlerValidationTest`, same applies to other tests.